src/tequila/simulators/simulator_cirq.py

```diff
@@ -157,5 +214,15 @@ def update_variables(self, variables):
         else:
             self.resolver=None
 
+def retrieve_cirq_device(device):
+    if isinstance(device,str):
+        return cirq_devices[device.lower()]
```

Comment:
You can do auto lookup here if you want, thats easier to maintain than hardcoded dictionaries.